src/api/teams/helpers/sync-teams-validation-schema.js

```diff
@@ -0,0 +1,29 @@
+import Joi from 'joi'
+
+const syncTeamsValidationSchema = Joi.object({
+  teams: Joi.array()
+    .items(
+      Joi.object({
+        teamId: Joi.string()
+          .max(53)
```

Comment:
what does `53` mean here? might be worth sticking that in a named `const` if that helps explain things?

Reply:
max teamname length. Not sure why its 53, but i've centralised the validators